### ISSUE TITLE
Office of Disabilities Department Name Change

### ISSUE THREAD

**Issue by curtiskphsc** [phsc-edu/2018#308]
This department has made a name change. They have been going through their content and making the necessary edits, and I have been following up on my end as well. The new department name is
Student Accessibility Services
The sub domain should reflect this. The "home" link should be named "Student Accessibility Services" and the search link should be "Search Accessibility Services"

![accessibility](https://user-images.githubusercontent.com/28155245/43090845-89b2de8c-8e76-11e8-8527-1fc411ccd7b5.PNG)

**Comment by C13L0** [comment]
@curtiskphsc Will there be a change to the subdomain name?

**Comment by C13L0** [comment]
@curtiskphsc Requested changes have been made. Will there be a change to the subdomain?

**Comment by C13L0** [comment]
It isn't difficult on my end, I would only need to input a ticket with Network Services so they can create the new domain and update the dns. Once completed, I would have to update the Accessibility link in all website footers.

**Comment by curtiskphsc** [comment]
Another question. All of the department job titles on this page are now incorrect https://disabilities-services.phsc.edu/department-information
Do we have to change them manually?

**Comment by C13L0** [comment]
Their job titles should automatically update on login since we are pulling directly from Active Directory. There is not a way to change any of the information manually.

**Comment by curtiskphsc** [comment]
So they need to login to the sub domain in order for their titles to update? I just need to be very clear with them because they put the "job title updates" on a list of things that need to be edited on their sub domain in that email you were copied on as well.
"Office of Disabilities needs to be changed to Student Accessibility Services under all of our titles
Lisa Henkaline's title need to be changed to Student Accessibility Services Assistant
My title needs to be updated from Coordinator of Student Accessibility Services to Director of Student Accessibility Services"
In the directory this information is correct 
https://info.phsc.edu/course-schedule/bio/darcy-mcdowell
https://info.phsc.edu/course-schedule/bio/lisa-henkaline
https://info.phsc.edu/course-schedule/bio/nicole-bilodeau-mckay
but on the department information page it is incorrect
https://disabilities-services.phsc.edu/department-information

**Comment by C13L0** [comment]
The main directory at info.phsc.edu/directory pulls directly from HR.

Subdomain directories get pulled directly from Active Directory. In order for the subdomain to update, they will need to login with their PHSC username and password.

**Comment by curtiskphsc** [comment]
Roger that.

**Comment by curtiskphsc** [comment]
@C13L0 UPDATE: Awaiting comments from subdomain owner RE changing the domain name.

**Comment by curtiskphsc** [comment]
@C13L0 @boettcc @longc77 
Domain name change approved by Dean Henderson.
New domain name to be accessibility-services.phsc.edu
Please execute at your leisure.

Yes, you can change the domain name.

Chiquita A. Henderson, MPA, MS
Dean of Student Affairs and Enrollment Management
Pasco-Hernando State College
727-816-3205

Message sent from mobile device.
 
From: Darcy McDowell <mcdowed@phsc.edu>
Sent: Thursday, August 2, 2018 3:59 PM
To: Chiquita Henderson
Cc: Kevin Curtis
Subject: Domain Change
 
Good Afternoon Chiquita,

      I was able to upload the last of the forms and do another once over to make sure that I updated everything on my end, with Kevin's help. Would we be able to get the go-ahead to change the domain name?

Kind Regards,

Darcy McDowell
Director of Student Accessibility Services
Pasco-Hernando State College
(727) 816-3240
DM

**Comment by C13L0** [comment]
@curtiskphsc 

From: Rick Sanders
Sent: Tuesday, August 7, 2018 09:16 AM
To: Webmaster
Subject: Re: New url for Disabilities Services
 
I've made the DNS changes.  It may take some time for the changes to fully propagate.



Thanks,
 
Rick Sanders
Network Security Manager
Pasco-Hernando State College
10230 Ridge Road
New Port Richey, FL 34654
727-816-3472

**Comment by C13L0** [comment]
Update website footer link

- [ ]  Academic Success Center 
- [ ]  Admissions 
- [ ]  Advising 
- [ ]  Athletics
- [ ]  Career Services
- [ ]  Educator Preparation Institute
- [ ]  eFaculty Support
- [ ]  Equity Services
- [ ]  eStudent Support
- [ ]  Financial Aid
- [ ]  Financial Services
- [ ]  Foundation
- [ ]  Legal Department
- [ ]  Marketing
- [ ]  Office of Disabilities
- [ ]  Safety and Security
- [ ]  Student Life
- [ ]  Testing Services
- [ ]  Writing Center

**Comment by C13L0** [comment]
Completed
